src/theme/components/featuredContent.ts

```diff
@@ -46,37 +47,53 @@ const FeaturedContent = defineMultiStyleConfig({
             bgColor: "dark.ui.bg.default",
           },
           img: screenreaderOnly(),
+          containerType: "inline-size",
+          "@container (min-width: 0px)": {
+            "[data-wrapper]": {
+              flexDirection: imageAtEnd ? "column-reverse" : "column",
+            },
+            "[data-image]": {
+              height: "320px",
+              width: "100%",
+            },
+          },
+          [`@container (min-width: ${breakpoints.md})`]: {
```

Comment:
Similar observation as the Card at 600px, but I think the container query update looks better. We should just flag that we expect this to look different.

<img width="1289" alt="Screenshot 2025-01-29 at 9 30 48 PM" src="https://github.com/user-attachments/assets/0ecf4530-bc09-4ff9-af75-67a10d5cff98" />

Reply:
Yes there will be visual breaking changes since its against the container's width instead of the screen's width. The storybook padding causes the styles to update earlier at a larger screen width.